TDP-3395: Query recordings docs (#273)
Bandwidth/bandwidth.github.io: voice/methods/calls/getRecordings.md

{% method %}

## Retrieve Account Recordings

> Retrieve information about your 1000 least recent recordings.

Reviewer: I don't know if I like saying your 1000 least recent recordings here.  You can get more than that if we add minStartTime and maxStartTime parameters.  It would probably be better to say that it returns a max of 1000 recordings, sorted by startTime from oldest to newest.  Or something like that.

Reviewer 2: Agree, copypasta 🍝

### Request URL

> <code class="post">GET</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/recordings`

Reviewer: class=get

Reviewer 2: done

#### Basic Authentication

Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.

---

### Supported Parameters

| Parameter          | Description                                                                                          | Mandatory |
|:-------------------|:-----------------------------------------------------------------------------------------------------|:----------|
| from               | Filter results by the `from` field.                                                                  | No        |
| to                 | Filter results by the `to` field..                                                                   | No        |

> | minStartTime       | Filter results which have `startTime` greater than or equal `minStartTime` (in ISO8601 format).      | No        |

Author: ```suggestion
| minStartTime       | Filter results to recordings which have a `startTime` after or including `minStartTime` (in ISO8601 format).      | No        |
```

> | maxStartTime       | Filter results which have `startTime` lower than `maxStartTime` (in ISO8601 format).                 | No        |

Author: ```suggestion
| maxStartTime       | Filter results to recordings which have a `startTime` before `maxStartTime` (in ISO8601 format).                 | No        |
```
